cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -1069,12 +1113,17 @@ public:
 
             for (int channel = 0; channel < NUM_ACTIVE_CHANNELS; ++channel)
             {
-                privatized_decode_op[channel].Init(num_output_levels[channel],
-                                                   upper_level[channel],
-                                                   lower_level[channel]);
+              cudaError_t error = privatized_decode_op[channel].Init(num_output_levels[channel],
+                                                                     upper_level[channel],
+                                                                     lower_level[channel]);
 
-                if (num_output_levels[channel] > max_levels)
-                    max_levels = num_output_levels[channel];
+              if (error != cudaSuccess)
```

Comment:
Do you think `CubDebug` might be helpful here?

Reply:
Added `CubDebug`.